history-of-the-quakers/original/06-book-1-chapter-3.adoc

```diff
@@ -67,13 +76,16 @@ or until they shall find sufficient Security to be of good Behaviour,
 or be thence delivered by Order from ourselves.
 Hereof you are not to fail.
 
+[.signed-section-closing]
```

Comment:
I wasn't sure to do this as a SSCC or a SSCL...or something else. :)

Reply:
Yeah, this one could go either way.  Did Jason give you the "75%" rule? I would apply something like that here -- in general, a SSC should not be longer than about 75% of a line.  If it is longer than that, and it's not a SSCC (which this is not) then it's perfectly fine to just leave it unmarked as a normal paragraph.  I think that's what I would do here, since it's a bit longer than 75% of the line. 👍